src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java

```diff
@@ -141,7 +141,9 @@ public static void rect(Fluid fluid, int left, int top, int width, int height, i
 	 * Draws a beveled, round rectangle that is substantially similar to default Minecraft UI panels.
 	 */
 	public static void drawGuiPanel(int x, int y, int width, int height) {
-		drawGuiPanel(x, y, width, height, 0x555555, 0xC6C6C6, 0xFFFFFF, 0x000000);
+		// For the purposes of Java's weird love-hate relationship with unsigned integers,
+		// -1 functions as 0xFFFFFFFF - full alpha + full white.
+		drawGuiPanel(x, y, width, height, 0xFF555555, 0xFFC6C6C6, -1, 0xFF000000);
```

Comment:
-1 is convenient shorthand in code, but for an API module like this I'd go for the longhand `0xFFFFFFFF`.

Reply:
IntelliJ _was_ yelling at me for that, but it stopped? I dunno...fixed it all the same, sorry about that